docs/reference/commandline/build.md

```diff
@@ -457,3 +459,115 @@ space.
 storing two copies of the image, one for the build cache with all the cache
 layers in tact, and one for the squashed version.
 
+#### Pre-Requisites
+
+The example on this page is using experimental mode in Docker 1.13 that can be enabled by using an experimental flag or option to Docker daemon, users can start the Docker daemon with or without experimental features.
+
+By default, experimental flag is turned off. To see the experimental flag, please check Docker version.
+
+```
+Server:
+ Version:      1.13.1
+ API version:  1.26 (minimum version 1.12)
+ Go version:   go1.7.5
+ Git commit:   092cba3
+ Built:        Wed Feb  8 06:35:24 2017
+ OS/Arch:      linux/amd64
+ Experimental: false
+
+```
+To turn on experimental mode, Docker daemon needs to be restarted with experimental flag turned on.
+
+#### Experimental flag in Ubuntu 14.04:
+
+For Ubuntu 14.04, Docker daemon options are specified as part of upstart system manager. Users can enable experimental mode in Ubuntu 14.04 by changing /etc/default/docker:
```

Comment:
I think we are trying to de-emphasize use of these platform-specific config files in favor of `daemon.json`. @thaJeztah ?